server/server.go

```diff
@@ -373,3 +383,40 @@ func (cs *cribbageServer) ginPostAction(c *gin.Context) {
 
 	c.String(http.StatusOK, `action handled`)
 }
+
+// GET /suggest/hand?dealt=<cards>
+func (cs *cribbageServer) ginGetSuggestHand(c *gin.Context) {
+
+	hand, err := convertToHand(c.Query(`dealt`))
+	if err != nil {
+		c.String(http.StatusBadRequest, `Error: %s`, err)
+		return
+	}
+
+	sums, err := suggestions.GetAllTosses(hand)
+	if err != nil {
+		c.String(http.StatusBadRequest, `Error: %s`, err)
+		return
+	}
+
+	resp := network.ConvertToGetSuggestHandResponse(sums)
+	sort.Slice(resp, func(i, j int) bool {
+		return resp[i].HandPts.Avg > resp[j].HandPts.Avg
+	})
```

Comment:
Why is it better/worse to sort these on the backend as opposed to the frontend?

Reply:
:shrug: I don't know how to do it on the frontend yet :#